bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -42,7 +42,7 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-      - description: CommonService is the Schema for the commonservices API. Documentation For additional details regarding install parameters check https://ibm.biz/icpfs39install. License By installing this product you accept the license terms https://ibm.biz/icpfs39license 
```

Comment:
Just reminder, we need to keep this description. They are required by ibm certification

Reply:
sure, I will bring them back